lib/grpfp.gi

```diff
@@ -4935,7 +4936,17 @@ local iso,k,id,f;
   # first try whether the group is ``small''
   iso:=FPFaithHom(fam);
   if iso<>fail and Size(Image(iso))<50000 then
-    k:=Image(iso);
+    k:=ImagesSource(iso);
+  #return function(w)
+  #  if not w in FreeGroupOfFpGroup(Source(iso)) then Error("flasch");fi;
+  #  w:=ElementOfFpGroup(fam,w);
+  #  Print("wa=",w,"\n");
+  #  w:=ImageElm(iso,w);
+  #  Print("wb=",w,"\n");
+  #  w:=Factorization(k,w);
+  #  Print("wc=",w,"\n");
+  #  return UnderlyingElement(w);
+  #end;
```

Comment:
Also this change is introduced by the commit adding `ProcessEpimorphismToNewFpGroup` -- another accident?

Reply:
That is (commented out) debugging code in case an error arises later. I'll look before merging whether it is worth keeping.

Reply:
OK, fair enough